### ISSUE TITLE
FR: Offer an option to customize the location of stored metadata

### ISSUE THREAD

**Issue by UlyssesZh** [koreader/koreader#9751]
By default, KOReader stores metadata in the same folder as the book, but I don't want KOReader to pollute the folder. I hope there is a way for me to customize the location of stored metadata.

**Comment by pazos** [comment]
Duplicate of https://github.com/koreader/koreader/issues/9265

**Comment by poire-z** [comment]
>  I don't want KOReader to pollute the folder

KOReader does not pollute, KOReader is good and gentle, it is the most kind, it does what's best for you :)
But no, no way yet. See https://github.com/koreader/koreader/issues/9265#issuecomment-1168354597
I thought we could propose a user patch for that (as we already provided some patch previously). Tried quickly making one, which would do as the old one (save docsettings in koreader/history/), but it would need to clean up the old one - and there's still many things that would not work as well as before (ie. hint for opened books), so not spending more time on this.
So, _do not_ use that:
```lua
local DocSettings = require("docsettings")
DocSettings.ensureSidecar = function() end
local orig_flush =  DocSettings.flush
function DocSettings:flush()
    self.sidecar_file = nil
    orig_flush(self)
end
```
